transformer_engine/common/fused_attn/fused_attn_max_512.cu

```diff
@@ -0,0 +1,1555 @@
+/*************************************************************************
+ * Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ *
+ * See LICENSE for license information.
+ ************************************************************************/
+
+#include "fused_attn_max_512.h"
+
+#include <cuda_bf16.h>
+#include <cuda_fp16.h>
+#include <map>
+#include <unordered_map>
+#include <vector>
```

Comment:
Some of these headers are already in ../common.h

Reply:
Remove `unordered_map` in https://github.com/NVIDIA/TransformerEngine/pull/175/commits/b5aec2c5be5f83ea9bd44e299c742eae11d14ddc